src/viam/sdk/tests/mocks/camera_mocks.hpp

```diff
@@ -4,9 +4,9 @@
 #include <viam/api/component/camera/v1/camera.grpc.pb.h>
 #include <viam/api/component/camera/v1/camera.pb.h>
 
-#include <viam/sdk/components/camera/camera.hpp>
-#include <viam/sdk/components/camera/client.hpp>
-#include <viam/sdk/components/camera/server.hpp>
+#include <viam/sdk/components/camera.hpp>
+#include <viam/sdk/components/private/camera_client.hpp>
+#include <viam/sdk/components/private/camera_server.hpp>
```

Comment:
Yes. I want to understand whether tests / mocks actually require access to the concrete `FooServer` and `FooClient` types: it would be nice if they don't, because then we can actually not only make them "header private",  by not installing the headers, but we can make those types have hidden visibility so they aren't even part of the symbol table for the SDK library.